lightbulb/app.py

```diff
@@ -1262,13 +1262,15 @@ def flatten_command_option(
             await event.interaction.create_response([])
             return
 
-        def convert_response_value(val: t.Union[str, int, float, hikari.CommandChoice]) -> hikari.CommandChoice:
+        def convert_response_value(
+            val: t.Union[str, int, float, hikari.impl.AutocompleteChoiceBuilder]
+        ) -> hikari.impl.AutocompleteChoiceBuilder:
             if isinstance(val, (str, int, float)):
-                return hikari.CommandChoice(name=str(val), value=val)
+                return hikari.impl.AutocompleteChoiceBuilder(name=str(val), value=val)
             return val
```

Comment:
Should maybe consider converting `CommandChoice` to `AutocompleteChoiceBuilder` here too and throwing a warning that using `CommandChoice` is deprecated instead of entirely removing the functionality as it will likely impact a few users